test-clients/src/main/java/com/hedera/services/bdd/suites/autorenew/GracePeriodRestrictionsSuite.java

```diff
@@ -79,17 +80,17 @@ public static void main(String... args) {
     public List<HapiApiSpec> getSpecsInSuite() {
         return List.of(
                 new HapiApiSpec[] {
-                    gracePeriodRestrictionsSuiteSetup(),
-                    contractCallRestrictionsEnforced(),
-                    payerRestrictionsEnforced(),
-                    cryptoTransferRestrictionsEnforced(),
-                    tokenMgmtRestrictionsEnforced(),
-                    cryptoAndContractDeleteRestrictionsEnforced(),
-                    treasuryOpsRestrictionEnforced(),
-                    tokenAutoRenewOpsEnforced(),
-                    topicAutoRenewOpsEnforced(),
-                    cryptoUpdateRestrictionsEnforced(),
-                    gracePeriodRestrictionsSuiteCleanup(),
+                    //                    gracePeriodRestrictionsSuiteSetup(),
+                    contractCallRestrictionsEnforced()
+                    //                    payerRestrictionsEnforced(),
+                    //                    cryptoTransferRestrictionsEnforced(),
+                    //                    tokenMgmtRestrictionsEnforced(),
+                    //                    cryptoAndContractDeleteRestrictionsEnforced(),
+                    //                    treasuryOpsRestrictionEnforced(),
+                    //                    tokenAutoRenewOpsEnforced(),
+                    //                    topicAutoRenewOpsEnforced(),
+                    //                    cryptoUpdateRestrictionsEnforced(),
+                    //                    gracePeriodRestrictionsSuiteCleanup(),
                 });
```

Comment:
Are these tests commented for test purposes or are affected by the change?

Reply:
Forgot to uncomment them. FIXED now